aws/app/lambda/reliability/lib/s3FileInput.js

```diff
@@ -0,0 +1,89 @@
+const {
+  S3Client,
+  GetObjectCommand,
+  CopyObjectCommand,
+  DeleteObjectCommand,
+} = require("@aws-sdk/client-s3");
+
+const s3Client = new S3Client({ region: process.env.REGION });
+
+const environment = process.env.ENVIRONMENT || "staging";
+const reliabilityBucketName = `forms-${environment}-reliability-file-storage`;
+const vaultBucketName = `forms-${environment}-vault-file-storage`;
```

Comment:
@bryan-robitaille @Moro-Code I updated this to dynamically set the environment based on an environment var passed to the Lambda.  Previously this was hard-coded as `staging`.